packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -305,7 +305,12 @@ function isSafeUse(node: TSESTree.Node): boolean {
       return ['instanceof', '==', '!=', '===', '!=='].includes(parent.operator);
 
     case AST_NODE_TYPES.AssignmentExpression:
-      return parent.operator === '=' && node === parent.left;
+      return (
+        parent.operator === '=' &&
+        (node === parent.left ||
+          (node.type === AST_NODE_TYPES.MemberExpression &&
+            node.object.type === AST_NODE_TYPES.Super))
```

Comment:
Actually - I think we need to do a little more with this check.

This check will allow:
```ts
let x;
x = super.foo;
```

Which is obviously not correct.
We need to ensure that the left is a `this` member expression and that the right is a `super` member expression